blackout/blackout.php

```diff
@@ -78,6 +78,8 @@ function blackout_redirect ($a, $b) {
 		Logger::log('redirecting user to blackout page');
 		System::externalRedirect($myurl);
 	}
+
+	DI::l10n()->t('BLAA');
```

Comment:
As an inside joke, we should stick to "blubb" and ask annando on the next hackathon in person why ;-)